cfgmgr/buffermgrdyn.cpp

```diff
@@ -1010,15 +1060,24 @@ task_process_status BufferMgrDynamic::doRemovePgTask(const string &pg_key, const
 
     SWSS_LOG_NOTICE("Remove BUFFER_PG %s (profile %s, %s)", pg_key.c_str(), bufferPg.running_profile_name.c_str(), bufferPg.configured_profile_name.c_str());
 
-    // recalculate pool size
+    // Recalculate pool size
```

Comment:
can you have space on each // ? sometimes you have, sometimes you dont

Reply:
All fixed